test/functional/feature_assumeutxo.py

```diff
@@ -12,6 +12,9 @@
 ## Possible test improvements
 
 - TODO: test submitting a transaction and verifying it appears in mempool
+- TODO: the "assumed" field for a confirmed wallet transaction should be
+        present during background sync, start false and become true as
+        sync progresses
```

Comment:
82f19c7e97281bbd66a25e53f30fe2f80a4f9d11: such a test seems hard to write without a (background sync aware) `stopatheight` RPC. So far I just tested manually that the value is correct: absent before loading a snapshot, `true` while background sync is below it, `false` once background sync passed the confirmation height.